cmd/kms-router.go

```diff
@@ -63,7 +63,6 @@ func registerKMSRouter(router *mux.Router) {
 		kmsRouter.Methods(http.MethodGet).Path(version + "/key/status").HandlerFunc(gz(httpTraceAll(kmsAPI.KMSKeyStatusHandler)))
 
 		// KMS Policy APIs
-		kmsRouter.Methods(http.MethodPost).Path(version+"/policy/set").HandlerFunc(gz(httpTraceAll(kmsAPI.KMSSetPolicyHandler))).Queries("policy", "{policy:.*}")
```

Comment:
We do not want to provide this API anymore? @aead

Reply:
Yes, this was pointless anyway. 

See commit message:

> The SetPolicy handler got removed since it may not get implemented by KES at all and could
not have been used in the past since stateless KES is read-only w.r.t. policies and identities.